code/concepts/concepts.cpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <list>
+#include <vector>
+
+/*
+
+In the code below, we provide two implementations of the function increment().
+The commented one is optimized for iterators which fulfill the requirements of
+the concept RandomAccessIterator.
+
+1. Uncomment and complete the concept RandomAccessIterator: check that the
+   class IterT has operator +=.
+2. Find an existing concept in the standard library, which can replace the
+   previous user-defined concept.
+
+*/
+
+/*
+
+template <typename IterT>
+concept RandomAccessIterator = requires(... TO BE COMPLETED...) {
+  ... TO BE COMPLETED...
+};
```

Comment:
Maybe point out that this concept is not meant to be a full random access iterator:
```suggestion
template <typename IterT>
concept RandomAccessIterator = requires(... TO BE COMPLETED...) {
  ... TO BE COMPLETED...
  (Only add what is necessary for this example)
};
```